tests/restart/15-state-to-db/cylc-suite-db.dump

```diff
@@ -0,0 +1,117 @@
+INSERT INTO "task_job_logs" VALUES('20020101T0000Z','foo',1,'job.status','20020101T0000Z/foo/01/job.status','1467285891.14442',236);
+INSERT INTO "task_job_logs" VALUES('20020101T0000Z','foo',1,'job.out','20020101T0000Z/foo/01/job.out','1467285891.47443',502);
+INSERT INTO "task_job_logs" VALUES('20020101T0000Z','foo',1,'job.err','20020101T0000Z/foo/01/job.err','1467285890.42243',0);
+INSERT INTO "task_job_logs" VALUES('20020101T0000Z','foo',1,'job-activity.log','20020101T0000Z/foo/01/job-activity.log','1467285891.26045',178);
+INSERT INTO "task_job_logs" VALUES('20010101T0000Z','bar',1,'job','20010101T0000Z/bar/01/job','1467285891.26944',4489);
+INSERT INTO "task_job_logs" VALUES('20010101T0000Z','bar',1,'job.status','20010101T0000Z/bar/01/job.status','1467285892.16246',236);
+INSERT INTO "task_job_logs" VALUES('20010101T0000Z','bar',1,'job.out','20010101T0000Z/bar/01/job.out','1467285892.49644',502);
+INSERT INTO "task_job_logs" VALUES('20010101T0000Z','bar',1,'job.err','20010101T0000Z/bar/01/job.err','1467285891.44442',0);
+INSERT INTO "task_job_logs" VALUES('20010101T0000Z','bar',1,'job-activity.log','20010101T0000Z/bar/01/job-activity.log','1467285892.31045',178);
+INSERT INTO "task_job_logs" VALUES('20030101T0000Z','foo',1,'job','20030101T0000Z/foo/01/job','1467285892.32047',4489);
+INSERT INTO "task_job_logs" VALUES('20030101T0000Z','foo',1,'job.status','20030101T0000Z/foo/01/job.status','1467285893.20646',236);
+INSERT INTO "task_job_logs" VALUES('20030101T0000Z','foo',1,'job.out','20030101T0000Z/foo/01/job.out','1467285893.52746',502);
+INSERT INTO "task_job_logs" VALUES('20030101T0000Z','foo',1,'job.err','20030101T0000Z/foo/01/job.err','1467285892.48846',0);
+INSERT INTO "task_job_logs" VALUES('20030101T0000Z','foo',1,'job-activity.log','20030101T0000Z/foo/01/job-activity.log','1467285893.36046',178);
+INSERT INTO "task_job_logs" VALUES('20020101T0000Z','bar',1,'job','20020101T0000Z/bar/01/job','1467285893.36946',4489);
+INSERT INTO "task_job_logs" VALUES('20020101T0000Z','bar',1,'job.status','20020101T0000Z/bar/01/job.status','1467285894.25348',236);
+INSERT INTO "task_job_logs" VALUES('20020101T0000Z','bar',1,'job.out','20020101T0000Z/bar/01/job.out','1467285894.59348',502);
+INSERT INTO "task_job_logs" VALUES('20020101T0000Z','bar',1,'job.err','20020101T0000Z/bar/01/job.err','1467285893.54145',0);
+INSERT INTO "task_job_logs" VALUES('20020101T0000Z','bar',1,'job-activity.log','20020101T0000Z/bar/01/job-activity.log','1467285894.41152',178);
+INSERT INTO "task_job_logs" VALUES('20030101T0000Z','bar',1,'job','20030101T0000Z/bar/01/job','1467285895.46549',4489);
+INSERT INTO "task_job_logs" VALUES('20030101T0000Z','bar',1,'job.status','20030101T0000Z/bar/01/job.status','1467285896.35151',236);
+INSERT INTO "task_job_logs" VALUES('20030101T0000Z','bar',1,'job.out','20030101T0000Z/bar/01/job.out','1467285896.66652',502);
```

Comment:
I think we do need to think about ditching this information, either by housekeeping or just not storing it...